cli/args/flags.rs

```diff
@@ -1756,7 +1756,6 @@ These must be added to the path manually if required.")
         Arg::new("root")
           .long("root")
           .help("Installation root")
-          .value_parser(value_parser!(PathBuf))
```

Comment:
I don't like this because it gives a false sense of what kind of data this is.

Reply:
I agree